internal/cmd/gocdk/demo_test.go

```diff
@@ -424,25 +424,30 @@ func TestAddDemo(t *testing.T) {
 		{
 			demo:        "secrets",
 			description: "encrypt works",
-			urlPaths:    []string{"/demo/secrets/encrypt"},
-			op:          "GET",
-			urlQuery:    "plaintext=my-sample-plaintext",
+			urlPaths:    []string{"/demo/secrets/", "/demo/secrets/encrypt"},
```

Comment:
I added `/demo/secrets/` as a path to test here, because the encryption handler is registered for it as well as for `/demo/secrets/encrypt`.